dcos/dcos-ui
# DCOS-49106: Nodes table showing infinite loading when filter doesn't find matches

Remove the loader in case no data is found.

Closes https://jira.mesosphere.com/browse/DCOS-49106

## Testing
1. Go to the nodes table in the nodes tab
2. Filter by ip that doesn't exist
3. Ensure that the table doesn't load forever

## Trade-offs
1. We need to decide how to distinguish between no data, because it is loading and not data because no data is found.
2. We need to decide what to show when no data is found.

## Dependencies
None.

## Screenshots
### Before
![screenshot from 2019-03-05 12-52-58](https://user-images.githubusercontent.com/40791275/53800539-a7adf680-3f45-11e9-817c-9f9b4c49a216.png)

### After
![screenshot from 2019-03-05 12-49-18](https://user-images.githubusercontent.com/40791275/53800450-6cabc300-3f45-11e9-9164-45b526817976.png)
